composer.json

```diff
@@ -88,25 +69,43 @@
   },
   "scripts": {
     "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
-    "create-required-files": "SkilldDrupal\\composer\\ScriptHandler::createRequiredFiles",
     "list-scaffold-files": [
       "SkilldDrupal\\composer\\ScriptHandler::listScaffoldFiles"
     ],
     "pre-install-cmd": [
       "SkilldDrupal\\composer\\ScriptHandler::checkComposerVersion"
     ],
+    "create-required-files": "SkilldDrupal\\composer\\ScriptHandler::createRequiredFiles",
     "pre-update-cmd": [
       "SkilldDrupal\\composer\\ScriptHandler::checkComposerVersion"
     ]
   },
   "extra": {
     "installer-paths": {
-      "web/core": ["type:drupal-core"],
-      "web/libraries/{$name}": ["type:drupal-library"],
-      "web/modules/contrib/{$name}": ["type:drupal-module"],
-      "web/profiles/contrib/{$name}": ["type:drupal-profile"],
-      "web/themes/contrib/{$name}": ["type:drupal-theme"],
-      "drush/contrib/{$name}": ["type:drupal-drush"]
+      "web/core": [
+        "type:drupal-core"
+      ],
+      "web/libraries/{$name}": [
+        "type:drupal-library"
+      ],
+      "web/modules/contrib/{$name}": [
+        "type:drupal-module"
+      ],
+      "web/profiles/contrib/{$name}": [
+        "type:drupal-profile"
+      ],
+      "web/themes/contrib/{$name}": [
+        "type:drupal-theme"
+      ],
+      "drush/Commands/contrib/{$name}": [
+        "type:drupal-drush"
+      ],
+      "web/modules/custom/{$name}": [
+        "type:drupal-custom-module"
+      ],
+      "web/themes/custom/{$name}": [
+        "type:drupal-custom-theme"
+      ]
```

Comment:
AFAIK every drupal module or theme installed with composer is contrib so there won't be need for these lines